website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -17,14 +18,15 @@ This year, dbt Labs is introducing an expanded notion of `dependencies` across m
 
 ## Prerequisites
-- Available in [dbt Cloud Enterprise](https://www.getdbt.com/pricing). If you have an Enterprise account, you can unlock these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-write-cross-project-ref). <Lifecycle status="enterprise"/>
-- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). You need at least one successful job run after defining their `access`.
-- Define a deployment environment in the upstream ("producer") project [that is set to be your Production environment](/docs/deploy/deploy-environments#set-as-production-environment), and ensure it has at least one successful job run in that environment.
+- Available in [dbt Cloud Enterprise](https://www.getdbt.com/pricing). To use it, designate a [public model](/docs/collaborate/govern/model-access) and add a [cross-project ref](#how-to-write-cross-project-ref).
+- Upstream project setup:
+  - Define models in an upstream ("producer") project with [`access: public`](/reference/resource-configs/access). Have least one successful job run after defining `access`.
+- Define a deployment environment in the upstream ("producer") project [set as your Production environment](/docs/deploy/deploy-environments#set-as-production-environment). Have at least one successful job run in that environment.
 - If the upstream project has a Staging environment, run a job in that Staging environment to ensure the downstream cross-project ref resolves.
-- Each project `name` must be unique in your dbt Cloud account. For example, if you have a dbt project (codebase) for the `jaffle_marketing` team, you should not create separate projects for `Jaffle Marketing - Dev` and `Jaffle Marketing - Prod`. That isolation should instead be handled at the environment level.
-  - We are adding support for environment-level permissions and data warehouse connections; please contact your dbt Labs account team for beta access.
-- The `dbt_project.yml` file is case-sensitive, which means the project name must exactly match the name in your `dependencies.yml`.  For example, if your project name is `jaffle_marketing`, you should use `jaffle_marketing` (not `JAFFLE_MARKETING`) in all related files.
-
+- Each project `name` must be unique in your dbt Cloud account.
+  - For example, if you have a dbt project (codebase) for the `jaffle_marketing` team, avoid creating projects for `Jaffle Marketing - Dev` and `Jaffle Marketing - Prod`; use environment-level isolation instead.
+- dbt Cloud supports [configuring different data platform connections](/docs/cloud/connect-data-platform/about-connections#connection-management) per environment for a single project using Global connections. This removes the need to duplicate projects for different accounts.
```

Comment:
@ericbaumann -- is the feature name 'global connections'? i don't see it mentioned in the cloud ui and docs either so wanting to confirm to make any corrections 🙏